backends/dpdk/dpdkArch.h

```diff
@@ -842,33 +842,40 @@ class CopyMatchKeysToSingleStruct : public P4::KeySideEffect {
  * Common code between SplitActionSelectorTable and SplitActionProfileTable
  */
 class SplitP4TableCommon : public Transform {
+    cstring switchExprTmp;
+    DeclarationInjector injector;
+
  public:
     enum class TableImplementation { DEFAULT, ACTION_PROFILE, ACTION_SELECTOR };
     P4::ReferenceMap* refMap;
     P4::TypeMap* typeMap;
     DpdkProgramStructure* structure;
+    std::map<cstring, std::vector<std::tuple<cstring, IR::Constant*>>>& actionCaseMap;
```

Comment:
in general I try to document such complicated maps, explaining what the keys and values are.

Reply:
an alternative is to wrap this into a separate class

Reply:
Just to clarify, are you suggesting to put this actionCaseMap into a separate class?

Reply:
Yes